src/components/modals/SharingModal.js

```diff
@@ -75,6 +84,16 @@ const SharingModal = ({
                 <FaShareAlt size={15} />
               </Button>
             </OverlayTrigger>
+            
+          </div>
+          <hr />
+          <h3>QR Code</h3>
+          <Button className="icon-button" variant="primary" onClick={() => setShowQRCode(true)}>
+            Generate QR Code
+            <FaQrcode size={15} />
+          </Button>
+          <div className="qr-code">
+            {showQRCode && <QRCode value={inviteLink} />}
```

Comment:
...which is to say: please tweak this to say `{showQRCode && <QRCode value={getWindowLocationHref()} />}`